### PR TITLE
added governing-body classification

### PR DESCRIPTION
**Pull request by LBONTEN**
added classification of governing bodies so we can filter on these attributes in Lokaal Beslist
classification of administrative units still need to be added in the future
:

### CODE REVIEW

**Comment by LBONTEN** [comment]
added administrative unit classifications as well

File: config/search/config.json [hunk 1]
```diff
@@ -81,6 +81,13 @@
           "http://data.vlaanderen.be/ns/besluit#bestuurt",
           "http://mu.semte.ch/vocabularies/core/uuid"
         ],
+        "administrative_unit_classification": [
```
Comment (aliokan) [inline config/search/config.json:84]: **Blocker:** Do we need it? If yes, you should add the `abstract_` version too

**Review by aliokan — CHANGES_REQUESTED** [review]
@LBONTEN Have you tried locally? Isn't mapping definition needed?

File: config/search/config.json (hunk 1 above)
Comment (LBONTEN) [reply config/search/config.json:84]: Oh, there is no abstract_administrative_unit in the rest of the config. So I didnt add the classification either. 
And not at the moment, no. But I'm guessing we might soon as Senne is working on adding filters on classification.

Sadly I can't test locally (definitely something I need to look into more in the future). So I tested it on dev, which worked.

Comment (aliokan) [reply config/search/config.json:84]: It's a mistake, we can actually remove them.

**Pull request merged**